flow2ml/Flow.py

```diff
@@ -102,57 +117,66 @@ def applyAugmentation(self,operations):
+
     progress = [(100/len(operations))*i for i in range(0,len(operations)) ]
     progress_i = 0
 
     for folder in self.classes:
       for operation in self.operations:
-        path = os.path.join(self.dataset_dir ,self.data_dir, folder)
+        paths = [os.path.join(self.dataset_dir ,self.data_dir, folder)]
+        # if apply_on_filters is true, creates a list of folders of filtered images to apply the augmentations to
+        if apply_on_filters:
+          for filter in apply_on_filters:
+            paths = [os.path.join(self.dataset_dir ,self.data_dir, folder, filter.title() + "Images")]
         
-        if operation == "flip":
-          self.applyFlip(path)
+         # applies augmentations to all folders in paths (single folder if no apply_on_filters)
+        for path in paths:
+          if operation == "flipped":
+            self.applyFlip(path)
 
-        if operation == "rotate":
-          self.applyRotate(path)
-        
-        if operation == "shear":
-          self.applyShear(path)
+          if operation == "rotated":
+            self.applyRotate(path)
+          
+          if operation == "sheared":
+            self.applyShear(path)
 
-        if operation == "invert":
-          self.applyInvert(path)
+          if operation == "inverted":
+            self.applyInvert(path)
 
-        if operation == "Hist_Equal":
-          self.applyHistogramEqualization(path)
+          if operation == "histogramequalised":
+            self.applyHistogramEqualization(path)
+            
+          if operation == "CLAHE":
+            self.applyCLAHE(path)
           
-        if operation == "CLAHE":
-          self.applyCLAHE(path)
-        
-        if operation == "crop":
-          self.applyCrop(path)
+          if operation == "cropped":
+            self.applyCrop(path)
 
-        if operation == "scale":
-          self.applyScale(path)
+          if operation == "scaled":
+            self.applyScale(path)
 
-        if operation == "zoom":
-          self.applyZoom(path)
+          if operation == "zoomed":
+            self.applyZoom(path)
 
-        if operation == "greyscale":
-          self.applyGreyscale(path)
+          if operation == "greyscale":
+            self.applyGreyscale(path)
 
-        if operation == "erode":
-          self.applyErode(path)
+          if operation == "eroded":
+            self.applyErode(path)
 
-        if operation == "dilate":
-          self.applyDilate(path)
+          if operation == "dilated":
+            self.applyDilate(path)
 
-        if operation == "open":
-          self.applyOpen(path)
+          if operation == "opened":
+            self.applyOpen(path)
 
-        if operation == "close":
-          self.applyClose(path)
+          if operation == "closed":
+            self.applyClose(path)
 
-        if operation == "threshold":
-          self.applyThreshold(path)
+          if operation == "thresholded":
+            self.applyThreshold(path)
```

Comment:
Change formatting of line 181 and 182 too

Reply:
Done